en/02_Developer_Guides/08_Performance/00_Partial_Caching.md

```diff
@@ -163,6 +155,9 @@ All you need to do to swap the cache backend for partial template cache blocks i
 
 Here's an example of how it could be done:
 
+> [!NOTE]
+> For the below example to work it is necessary to have the Injector service `App\Cache\Service.memcached` defined somewhere in the configs.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.